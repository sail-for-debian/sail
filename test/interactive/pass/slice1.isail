:slice_roots s3
:slice
:ast out.sail
:quit
