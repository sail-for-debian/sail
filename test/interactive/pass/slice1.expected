default Order dec







































$sail_internal 






enum e2 = {GHI, JKL}

struct s1 = {fA : int, fB : e2}

struct s3 = {fE : s1, fF : int}
